cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -112,20 +112,11 @@ protected int getInstances(HttpRequest request) throws BadRequestException {
   }
 
   protected Map<String, String> decodeArguments(HttpRequest request) throws JsonSyntaxException {
-    ChannelBuffer content = request.getContent();
-    if (!content.readable()) {
+    Map<String, String> args = parseBody(request, STRING_MAP_TYPE);
```

Comment:
There's places calling this, unrelated to my code, so filed this JIRA for this task:
https://issues.cask.co/browse/CDAP-6296

My PR doesn't use this method; it simply cleaned up surrounding code.